tmt/schemas/common.yaml

```diff
@@ -268,3 +279,8 @@ definitions:
         pattern: "^/"
 
       - $ref: "/schemas/common#/definitions/fmf_id"
+
+  # In multihost scenarios, some steps can be executed on some guests only.
+  # https://tmt.readthedocs.io/en/stable/spec/plans.html#multihost
+  where:
+    type: string
```

Comment:
Should be a string or a list of strings. Here's a `prepare` config example:

```yaml
prepare:
  - how: ansible
    playbook: common.yaml
    where: [primary, replica]
```

Reply:
Ack, fixed in [a909cd0](https://github.com/teemtee/tmt/pull/1421/commits/a909cd01845b0c4a41466ebd32cd90a2e235592c).